docs/tutorial.ipynb

```diff
@@ -273,7 +273,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "IntegerType(parameters=[IntAttr(data=64)], width=IntAttr(data=64)) should be of base attribute int\n"
+      "In integer_type attribute verifier: 2 parameters expected, got 1\n"
```

Comment:
Do we want this change in this PR?

Reply:
It's the same output as we get on the CI, so I don't think there's much choice. Will need updating when #455 gets resolved